packages/block-library/src/cover/editor.scss

```diff
@@ -65,3 +65,42 @@
 	// Important is used to have higher specificity than the inline style set by re-resizable library.
 	height: auto !important;
 }
+
+// Only target direct dropzone.
+.wp-block-cover > .components-drop-zone {
+	&.is-active {
+		transition: 0.3s opacity, 0.3s border;
+	}
+
+	&.is-dragging-over-element {
+		background-color: transparent;
+		border: $grid-unit-60 solid var(--wp-admin-theme-color);
+
+		.components-drop-zone__content {
+			transform: none;
+		}
+	}
+
+	.components-drop-zone__content {
+		display: flex;
+		align-items: center;
+		top: -36px;
```

Comment:
I wonder if we should round this number up to 40px ($grid-unit-50), or down to 32 ($grid-unit-40). It's a small thing, but might be slightly more griddy. Alternately, I see the grid as being fundamentally base12, even though we leverage an 8px base. You could do `($grid-unit-15 * 3)`.

Reply:
Tested with both `$grid-unit-40` and `$grid-unit-40` initially, but 36px give it more "centered" look.

I've updated styles to use `($grid-unit-15 * 3)`.